bpf/kmesh/workload/include/backend.h

```diff
@@ -28,6 +28,8 @@ static inline int waypoint_manager(struct kmesh_context *kmesh_ctx, struct ip_ad
         value_tuple.ipv4.daddr = kmesh_ctx->orig_dst_addr.ip4;
         value_tuple.ipv4.dport = ctx->user_port;
     } else if (ctx->family == AF_INET6) {
+        if (!is_ipv4_mapped_addr(kmesh_ctx->orig_dst_addr.ip6))
```

Comment:
The return type is of type bool, so remove "!"?

Reply:
how can waypoint process ipv6(v4 mapped format)?

Reply:
waypoint could correctly handle pure ipv4, pure ipv6 and ipv4-mapped ipv6 address, ref: https://github.com/envoyproxy/envoy/blob/main/source/common/network/address_impl.cc#L87 

As long as the address encoded in tlv is valid.